docs/source/reference/router/telemetry/instrumentation/standard-instruments.mdx

```diff
@@ -127,4 +115,8 @@ The following metrics have been deprecated and should not be used.
 - `apollo_authentication_success_count` - **Deprecated**: use the `apollo.router.operations.authentication.jwt` metric instead. If the `authentication.jwt.failed` attribute is *absent* or `false`, the authentication succeeded.
 - `apollo_require_authentication_failure_count` - **Deprecated**: use the `http.server.request.duration` metric's `http.response.status_code` attribute. Requests with authentication failures have HTTP status code 401.
 - `apollo_router_timeout` - **Deprecated**: this metric conflates timed-out requests from client to the router, and requests from the router to subgraphs. Timed-out requests have HTTP status code 504. Use the `http.response.status_code` attribute on the `http.server.request.duration` metric to identify timed-out router requests, and the same attribute on the `http.client.request.duration` metric to identify timed-out subgraph requests.
-- `apollo_router_http_request_retry_total` **Deprecated**: use the `http.client.request.duration` metric's `http.request.resend_count` attribute. Requests with retries will contains `http.request.resend_count` set with the number of retries.
\ No newline at end of file
+- `apollo_router_session_count_active` - **Deprecated**: Number of in-flight GraphQL requests
+- `apollo_router_http_requests_total` - **Deprecated**: Total number of HTTP requests by HTTP status
+- `apollo_router_http_request_duration_seconds_bucket` - **Deprecated**: HTTP router request duration
+- `apollo_router_http_request_duration_seconds_bucket` - **Deprecated**: HTTP subgraph request duration, attributes:
+  - `subgraph`: (Optional) The subgraph being queried
```

Comment:
Can we add the replacements to each or explicitly say there is no replacement?

Reply:
Maybe I should add the link to the new instruments docs ?

Reply:
Maybe : ) IMO it's important that we have a clear list of "if using THIS old metric, you have to change to THIS"

Reply:
Done in https://github.com/apollographql/router/pull/6547/commits/11d5c984e3f5734109d03443aea5841bc9beb386